### ISSUE TITLE
Sidebar refuses to resize, shrinks, and gets stuck when resizing (Linux Mint 19)

### ISSUE THREAD

**Issue by ajstensland** [adobe/brackets#14487]
### Description and Reproduction Steps
When I try to resize the sidebar from the default size:
![brackets debug 2](https://user-images.githubusercontent.com/35818081/43019314-80533646-8c2a-11e8-9e88-aade0ec7b894.png)

It fails to resize to the specified dimension (if anything, it just grows by a few pixels):
![screenshot from 2018-07-20 15-54-21](https://user-images.githubusercontent.com/35818081/43020125-fbac1fcc-8c2c-11e8-88a3-21eb9831cf12.png)

When I click again to try again, the sidebar hides itself:
![brackets debug 5](https://user-images.githubusercontent.com/35818081/43019631-a51c3fe4-8c2b-11e8-8bbf-78761e41c9d4.png)

However, when I try to resize from here it either a) does nothing or b) opens the sidebar to a very small, unmanageable size:
![brackets debug 4](https://user-images.githubusercontent.com/35818081/43019851-3a6d5f38-8c2c-11e8-9af7-90b62b7686dc.png)

Sometimes, and I'm not sure when and when not, there is a small tab poking out (not sure if this is intended or not) like so:
![brackets debug 7](https://user-images.githubusercontent.com/35818081/43019802-233904ac-8c2c-11e8-8ac7-fb5d2faa4615.png)

I encountered this error on my first-ever install of Brackets within 30 seconds of use. See Attempted Fixes for more information.

### TL;DR

**Expected behavior:** For the sidebar's width to follow my cursor and stop resizing when the mouse button was lifted.

**Actual behavior:** The sidebar does not move with the mouse, and when the mouse button is lifted, the <-> resizing cursor remains visible wherever I put the mouse on the window. Then, clicking again on the sidebar border, the sidebar hides itself. After repeating some number of times, the sidebar, when opening, only opens by about 10-30 pixels and will not grow much more (size seems arbitrary when dragging open).

### Attempted Fixes
Restarting Brackets does not fix the problem -- the sidebar stays in its same position. 

Running with extensions disabled does not fix the problem.

Changing from my NVidia card to the integrated Intel graphics does not fix the problem.

However, deleting ~/.config/Brackets/cef-data, state.json, and brackets.json _does_ solve the problem _until_ I try to resize the sidebar again.

### Versions
OS: Linux Mint 19 Tara
Linux Kernel: Linux 4.15.0-24-generic
Brackets: 1.13.0-17696 (Installed from snap)

**Comment by D-Jones-1** [comment]
If there's anything I can do to fix the bug and someone can explain the work to be done, I'd be happy to.

**Comment by csrohit** [comment]
Check for the libcurl3 and libpango1.0-0 dependencies.
I tried to reinstall brackets and got these missing dependency errors. After installing the required dependencies Brackets is working fine

**Comment by ajstensland** [comment]
I just noticed that if I run Brackets through the console, I get the following errors: 
`[0921/113742:ERROR:browser_main_loop.cc(261)] Gtk: Locale not supported by C library.
	Using the fallback 'C' locale.
[0921/113743:ERROR:browser_main_loop.cc(261)] <unknown>: Pixbuf theme: Cannot load pixmap file /usr/share/themes/Mint-Y-Dark/gtk-2.0/menubar-toolbar/menubar-dark.png: Unable to load image-loading module: /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-png.so: /snap/core/current/lib/x86_64-linux-gnu/libz.so.1: version `ZLIB_1.2.9' not found (required by /usr/lib/x86_64-linux-gnu/libpng16.so.16)


(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_width: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_height: assertion 'GDK_IS_PIXBUF (pixbuf)' failed
[0921/113743:ERROR:browser_main_loop.cc(239)] GTK theme error: Invalid borders specified for theme pixmap:
        /usr/share/themes/Mint-Y-Dark/gtk-2.0/menubar-toolbar/menubar-dark.png,
borders don't fit within the image

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_n_channels: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_pixels_with_length: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_rowstride: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_n_channels: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_pixels_with_length: assertion 'GDK_IS_PIXBUF (pixbuf)' failed

(Brackets:8895): GdkPixbuf-CRITICAL **: gdk_pixbuf_get_rowstride: assertion 'GDK_IS_PIXBUF (pixbuf)' failed
`
I'm just a casual user -- I'm not very well versed on GTK and Linux libraries and stuff, but I hope this helps in some way.

**Comment by alsavov** [comment]
I got the same issue here under Ubuntu 18.04

**Comment by Khotso-t** [comment]
Just install brackets from the main repo

```

sudo add-apt-repository ppa:webupd8team/brackets
sudo apt update
sudo install brackets

```

**Comment by shubhsnov** [comment]
> [0921/113742:ERROR:browser_main_loop.cc(261)] Gtk: Locale not supported by C library.
> Using the fallback 'C' locale.

It seems like this issue is related to #14581 (comment), which can be fixed by installing the language pack in Ubuntu.

See my comment here: #14581 (comment)

**Comment by nagaomkar9** [comment]
I had the same issue and for me it seems to occur when the language is set to 'system default'. So I went to Debug -> Switch Language menu and choose english, and it solve the issue. I tried with several other languages and it works perfectly.

**Comment by xidanezx** [comment]
vim  ~/.config/Brackets/state.json +1569
change the size from whatever it is to 300 and open brackets again..

  "healthDataNotificationShown": true,
    "sidebar": {
        "visible": true,
        "size": 300
    },

**Comment by kevinmt19** [comment]
> I had the same issue and for me it seems to occur when the language is set to 'system default'. So I went to Debug -> Switch Language menu and choose english, and it solve the issue. I tried with several other languages and it works perfectly.
> This should surely work.
> happy coding

Omg thank you so much man

**Comment by khadijanazih** [comment]
> I had the same issue and for me it seems to occur when the language is set to 'system default'. So I went to Debug -> Switch Language menu and choose english, and it solve the issue. I tried with several other languages and it works perfectly.
> This should surely work.
> happy coding

Thanks So Much! That Saved My Life ^^

**Comment by Suneelvarma5fi** [comment]
> I had the same issue and for me it seems to occur when the language is set to 'system default'. So I went to Debug -> Switch Language menu and choose english, and it solve the issue. I tried with several other languages and it works perfectly.
> This should surely work.
> happy coding

LOVE YOU

**Comment by ltrottier** [comment]
> I had the same issue and for me it seems to occur when the language is set to 'system default'. So I went to Debug -> Switch Language menu and choose english, and it solve the issue. I tried with several other languages and it works perfectly.
> This should surely work.
> happy coding

Worked for me. Thanks a lot !
